### PR TITLE
Set consentLanguage without fetch request

### PR DESCRIPTION
**Pull request by ccartesonetrust**
Calling the changeLanguage method on the GVL class will trigger a new request, resulting in some 404 errors. There are a couple of open tickets related to this issue:

- Cant set consentLanguage with custom GVL #211
- GVL object is being constructed with the gvl json, and setting consentLanguage forces another fetch #405

We propose adding a boolean value to **SKIP** the fetch request.

### CODE REVIEW

File: modules/core/src/GVL.ts [hunk 1]
```diff
@@ -474,7 +474,9 @@ export class GVL extends Cloneable<GVL> implements VendorList {
 
         try {
 
-          await this.fetchJson(url);
+          if (!skipFetchCall) {
+            await this.fetchJson(url);
+          }
```
Comment (marco-prontera) [inline modules/core/src/GVL.ts:479]: I think we could support this directly in the `GVL` constructor by giving the language, perhaps putting the lang as a parameter of an object to be given as a second parameter to the `GVL` constructor. In this way future additions will not cause breaking changes.

My opinion: Imagine calling the `GVL` constructor and then having to call this method with `skipFetchCall` set to `true`, it doesn't seem to fit well. Maybe you might like what I proposed. I hope this thought can help

**Review by HeinzBaumann — CHANGES_REQUESTED** [review]
I agree with @marco-prontera. Adding a language in the GVL conctructor might be better. 
Also, please add a test case to test.

**Review by HeinzBaumann — APPROVED** [review]
Updates look good to me. @marco-prontera can you also take a quick look at the latest changes? Thanks

**Comment by 1arrow** [comment]
@marco-prontera can you take a look at the change pushed based on your comment?

**Review by marco-prontera — APPROVED** [review]
LGTM

**Comment by ccartesonetrust** [comment]
> LGTM

Hi @marco-prontera, do I have to do something else? What is the process for merging this code into the main branch?

**Comment by HeinzBaumann** [comment]
@lamrowena Can you hit the approve also and merge this. Marco and I reviewed and approved the changes.

**Review by lamrowena — APPROVED** [review]

**Comment by 1arrow** [comment]
When is the new release planned? we are waiting for this fix.

**Pull request merged**